app/controllers/Healthcheck.scala

```diff
@@ -1,12 +1,15 @@
 package controllers
 
-import actions.CommonActions.NoCacheAction
 import play.api.mvc._
+import services.TouchpointBackend
 
 object Healthcheck extends Controller {
 
-  def index = NoCacheAction {
-    Ok("""{"status":"OK"}""").withHeaders(CONTENT_TYPE -> "application/json")
+  def index = Action {
+    Cached(1) {
+      if(TouchpointBackend.Normal.zuoraService.products.size > 0) Ok("OK")
+      else ServiceUnavailable("Service Unavailable")
+    }
   }
 
```

Comment:
I've followed membership-frontend for the healthcheck. We need to ensure there are product plans fetched from Zuora or the user won't be able to subscribe